xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererGBM.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *      Copyright (C) 2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#include "cores/RetroPlayer/buffers/BaseRenderBufferPool.h"
+#include "cores/RetroPlayer/process/RPProcessInfo.h"
+#include "RPRendererOpenGLES.h"
+
+#include "system_gl.h"
+
+#include <atomic>
+#include <stdint.h>
+#include <vector>
```

Comment:
only `<memory> is needed. we can drop #includes for types in functions of `// implementation of <>` blocks because the base classes will pull in those headers.